src/main/java/org/reaktivity/nukleus/http2/internal/stream/Http2ServerFactory.java

```diff
@@ -817,7 +822,7 @@ private int decodeContinuation(
         return progress;
     }
 
-    private int decodeData(
+    private int decodeDataLength(
```

Comment:
This should just be `decodeData` as that is the "state" we are in when starting to decode the `H2_DATA` frame.